app/assets/stylesheets/provider/admin/_dashboard.scss

```diff
@@ -345,7 +345,7 @@
 }
 
 .DashboardNavigation {
-  flex: 1 1 0%; // Need to specify flex-basis for IE11 compatibility
```

Comment:
I think this comment was justified, since `flex: 1` is equivalent but won't work on IE11. And someone in the future could accidentally refactor and break it.

Reply:
Comment back